doi-transformer/src/test/java/no/unit/nva/doi/transformer/CrossRefConverterTest.java

```diff
@@ -334,8 +332,7 @@ public void toPublicationSetsTheMetadataSourceToTheCrossRefUrlWhenTheCrossrefDoc
     @Test
     @DisplayName("toPublication sets the MetadataSource to the specfied URL when the Crossref "
             + "document has as \"source\" a valid URL")
-    public void toPublicationSetsTheMetadataSourceToTheSourceUrlIfTheDocHasAsSourceAValidUrl()
-            throws InvalidIssnException, InvalidIsbnException, UnsupportedDocumentTypeException {
+    public void toPublicationSetsTheMetadataSourceToTheSourceUrlIfTheDocHasAsSourceAValidUrl() {
```

Comment:
JUnit5 package-private visibility

Reply:
package-private it is, as JUnit5 says...